timestamp-client/src/main/java/com/palantir/timestamp/RateLimitedTimestampService.java

```diff
@@ -40,7 +40,7 @@
  */
 @ThreadSafe
 public class RateLimitedTimestampService implements TimestampService {
```

Comment:
However moving RLTS to `atlasdb-config` is no good as well because large internal product has its own timestamp service, which pulls in `timestamp-impl` but not `atlasdb-config`. 

So the solution assuming we don't want to dev break or duplicate the class is either make a new project (what I did), or stick it in their common ancestor (`atlasdb-commons`) which I didn't really like either.